tap_outreach/sync.py

```diff
@@ -281,12 +281,7 @@ def sync_endpoint(client, config, catalog, state, start_date, stream, mdata):
                     filter_field)] = '{}..inf'.format(paginate_datetime)
                 query_params['sort'] = filter_field
 
-        LOGGER.info('{} - Syncing data since {} - page: {}, limit: {}, offset: {}'.format(
-            stream.tap_stream_id,
-            last_datetime,
-            page,
-            count,
-            offset))
+        LOGGER.info(f"{stream.tap_stream_id} - Syncing data since {last_datetime} - page: {page}, limit: {count}, offset: {offset}")
```

Comment:
Line is too long. Also f-string ideally should not be used inside loggers.

Reference: https://discuss.python.org/t/safer-logging-methods-for-f-strings-and-new-style-formatting/13802